autokeras/meta_model.py

```diff
@@ -21,13 +21,13 @@ def assemble(inputs, outputs, dataset):
     outputs = nest.flatten(outputs)
     assemblers = []
     for input_node in inputs:
-        if isinstance(input_node, hyper_node.TextInput):
+        if isinstance(input_node, node.TextInput):
             assemblers.append(TextAssembler())
-        if isinstance(input_node, hyper_node.ImageInput):
+        if isinstance(input_node, node.ImageInput):
             assemblers.append(ImageAssembler())
-        if isinstance(input_node, hyper_node.StructuredInput):
+        if isinstance(input_node, node.StructuredInput):
             assemblers.append(StructuredDataAssembler())
-        if isinstance(input_node, hyper_node.TimeSeriesInput):
+        if isinstance(input_node, node.TimeSeriesInput):
             assemblers.append(TimeSeriesAssembler())
```

Comment:
Why don't you put a class variable under `Input`? then you can do
```
assemblers.append(input_node.assembler())
```
it is not only saving you few lines of codes but also easier to manage the dependencies between `Input` and `Assembler`